packages/ffe-checkbox-react/src/Checkbox.js

```diff
@@ -17,16 +25,16 @@ export default function CheckBox(props) {
                 aria-invalid={String(invalid)}
                 {...rest}
             />
-            <label
-                className={classNames({
-                    'ffe-checkbox': true,
-                    'ffe-checkbox--inline': inline,
-                    'ffe-checkbox--no-margin': noMargins,
-                })}
-                htmlFor={id}
-            >
-                {label || children}
-            </label>
+            {
+                typeof children === 'function'
+                    ? children(labelProps)
+                    : (
+// eslint-disable-next-line jsx-a11y/label-has-for
+                        <label {...labelProps}>
```

Comment:
Optionally I could explicitly apply each of the labelProps to the label to avoid the eslint-disable-next-line comment.
Any thoughts?

Reply:
I think it's fine to just use the inline comment here, it clearly indicates why we're doing it.